### PR TITLE
Dashboard extra vis

### PR DESCRIPTION
**Pull request by david-nepo**
db_functions.py
- added st.cache_data decorator to data functions

config.toml
- added custom theme

Dockerfile
- added folder location for config.toml

main.py
- reformatted

### CODE REVIEW

File: dashboard/db_functions.py [hunk 1]
```diff
@@ -29,16 +29,17 @@ def get_database_engine():
         raise e
 
 
-def get_24hr_data(table_name: str, db_engine: db.Engine, connection: Connection, metadata,
+@st.cache_data(show_spinner='Plants be loading... 🌱')
```
Comment (caitlinturnidge) [inline dashboard/db_functions.py:32]: Nice cacheing!

File: dashboard/main.py [hunk 2]
```diff
@@ -118,21 +118,32 @@ def build_resample_rate_slider() -> str:
...
+    with col1:
+        latest_data = chart_data.iloc[-1]
+        st.subheader('🌡️ Temperature')
+
+    with col2:
+        st.metric(
+            f'Current Temperature (Plant {selected_plant})', f"{latest_data['temperature']:.1f}°C")
 
 
 def display_sidebar_image(selected_plant: int) -> None:
```
Comment (caitlinturnidge) [inline dashboard/main.py:149]: This look like a huge function, could probably be split up

Comment (caitlinturnidge) [inline dashboard/main.py:149]: This look like a huge function, could probably be split up

**Review by caitlinturnidge — APPROVED** [review]
Have seen the visualisations and I think they look great

File: dashboard/main.py (hunk 2 above)
Comment (Shurc1val) [reply dashboard/main.py:149]: I'm jumping on the file now, so will get working on that

**Pull request merged**